selfdrive/car/docs_definitions.py

```diff
@@ -21,6 +21,7 @@ class Column(Enum):
   STEERING_TORQUE = "Steering Torque"
   AUTO_RESUME = "Resume from stop"
   HARNESS = "Harness"
```

Comment:
I would combine these into one column, you can link the harness using the harness connector

Reply:
I merged the Harness and the Harness Content columns to get back to the original spacing